mm2src/coins/Cargo.toml

```diff
@@ -106,7 +106,8 @@ url = { version = "2.2.2", features = ["serde"] }
 uuid = { version = "1.2.2", features = ["fast-rng", "serde", "v4"] }
 # One of web3 dependencies is the old `tokio-uds 0.1.7` which fails cross-compiling to ARM.
 # We don't need the default web3 features at all since we added our own web3 transport using shared HYPER instance.
-web3 = { git = "https://github.com/KomodoPlatform/rust-web3", tag = "v0.19.0", default-features = false }
+# web3 = { git = "https://github.com/KomodoPlatform/rust-web3", tag = "v0.19.0", default-features = false }
+web3 = { git = "https://github.com/dimxy/rust-web3", default-features = false, branch = "add-chain-id" }
```

Comment:
You can use this release now https://github.com/KomodoPlatform/rust-web3/releases/tag/v0.20.0

Reply:
fixed [342d3f4](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2051/commits/342d3f4b8cfc50b9c4310ca262bd3314de0d2e9d)